hazelcast/src/main/java/com/hazelcast/client/impl/protocol/task/AbstractMessageTask.java

```diff
@@ -218,6 +213,7 @@ protected Connection findSendConnection() {
             return connection;
         }
 
+        String clientUuid = endpoint.getUuid();
```

Comment:
Do you mean this https://github.com/hazelcast/hazelcast/pull/9514/files#diff-62dfc5312563d71974e936132c44bd73L70 ?